nixos/doc/manual/release-notes/rl-2511.section.md

```diff
@@ -124,6 +124,8 @@
 
 - [Szurubooru](https://github.com/rr-/szurubooru), an image board engine inspired by services such as Danbooru, dedicated for small and medium communities. Available as [services.szurubooru](#opt-services.szurubooru.enable).
 
+- [LubeLogger](https://lubelogger.com/), a vehicle maintenance and fuel mileage tracker. Available as [services.lubelogger](option.html#opt-services.lubelogger).
```

Comment:
The release notes are full of non-compliant entries, so it's hard to get right. This is how it should look, following one-sentence-per-line:

```suggestion
- [LubeLogger](https://lubelogger.com/), a vehicle maintenance and fuel mileage tracker.
  Available as [services.lubelogger](option.html#opt-services.lubelogger).
```

But it's not _too_ important as they will be cleaned up by the release managers anyway.